server/binding.js

```diff
@@ -84,15 +91,21 @@ binding.query.Locus;
 /**
```

Comment:
Can we expand the definition of binding.query.Exons to replace {Object} on line79? Thanks.